admin/src/index.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+
 import { App } from './App';
 
+import './index.css';
```

Comment:
change `index.css` into `app.css` and use in `app.tsx`. let `index.tsx` be as entry point without any logic